Repository/AbstractPhpcrRepository.php

```diff
@@ -72,6 +82,58 @@ public function find($query, $language = 'glob')
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function remove($query, $language = 'glob')
+    {
+        $this->failUnlessGlob($language);
+        $nodes = $this->finder->find($this->resolvePath($query));
+
+        // if the query does not contain a glob pattern, then it applies to an
+        // explicit node and we should throw an exception if it is not found.
+        if (false === $this->globHelper->isGlobbed($query) && 0 === count($nodes)) {
+            throw new ResourceNotFoundException($query);
```

Comment:
Actually meant this as a coment on `move` -- I see that for `remove` the number of items is returned. We can simply return the number of nodes found by the finder.